Library/Formula/snakemake.rb

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
```

Comment:
This shouldn't be necessary. We have some background magic to interpret formulae files as UTF-8.